en/config_fw/weight_and_altitude_tuning.md

```diff
@@ -55,28 +66,28 @@ By calibrated we mean the value of that variable measured at sea level in standa
 
 E.g. by $\hat{\dot{h}}_{max}$ we specify the maximum climb rate the vehicle can achieve at [WEIGHT_BASE](../advanced_config/parameter_reference.md#WEIGHT_BASE) at sea level in standard atmospheric conditions.
 
-
 ### Effect of Weight on Maximum Climb Rate
 
-The maximum climb rate ([FW_T_CLMB_MAX](../advanced_config/parameter_reference.md#FW_T_CLMB_MAX)) must be scaled as a function of the weight ratio.
+The maximum climb rate ([FW_T_CLMB_MAX](../advanced_config/parameter_reference.md#FW_T_CLMB_MAX)) is scaled as a function of the weight ratio.
 
-From the steady state equations of motions of an airplane we find that the maximum climb rate can be written as
+From the steady state equations of motions of an airplane we find that the maximum climb rate can be written as:
 
 $$\dot{h}_{max} = { V * ( Thrust - Drag ) \over{m*g}}$$
 
 where `V` is the true airspeed and `m` is the vehicle mass.
-It's easy to see from this equation that the maximum climb rates scales with vehicle mass.
+From this equation we see that the maximum climb rates scales with vehicle mass.
 
 ### Effect of Weight on Minimum Sink Rate
 
-The minimum sink rate ([FW_T_SINK_MIN](../advanced_config/parameter_reference.md#FW_T_SINK_MIN)) must be scaled as a function of weight ratio
+The minimum sink rate ([FW_T_SINK_MIN](../advanced_config/parameter_reference.md#FW_T_SINK_MIN)) is scaled as a function of weight ratio
 
 The minimum sink rate can be written as:
 
 $$\dot{h}_{min} = \sqrt{2mg\over{\rho F}} f(c_A, c_W)$$
 
 where $\rho$ is the air density, F is the wing area and $f(c_A, c_W)$ is a function of the polars.
-It's easy to see that the minimum sink rate scales with the square root of the weight ratio.
+
+From this equation we see that the minimum sink rate scales with the square root of the weight ratio.
```

Comment:
FYI It is "easy to see", but what some see as easy is not as obvious to others. My training tells me to avoid suggesting things are easy.

Reply:
@hamishwillee I think I have read too many papers which suggest that something is "easy to see". I assume it's a way to try and make people feel dumb. I don't have a problem removing it. To me this section is for someone who is into the PX4 code and wants to double check calculations.

Reply:
That's precisely it - we don't want to make people feel dumb. I do think this is fairly easy to see, but it isn't a good habit to get into :-)